queries/python.scm

```diff
@@ -27,6 +27,58 @@
   (with_statement)
 ] @statement
 
+;;!! a = 25
+;;!      ^^
+;;!   xxxxx
+;;!  ------
+(assignment
+  (_) @_.leading.start.endOf
+  .
+  right: (_) @value @_.leading.end.startOf
+) @_.domain
+
+;;!! a /= 25
+;;!       ^^
+;;!   xxxxxx
+;;!  -------
+(augmented_assignment
+  (_) @_.leading.start.endOf
+  .
+  right: (_) @value @_.leading.end.startOf
+) @_.domain
+
+;;!! d = {"a": 1234}
+;;!            ^^^^
+;;!          xxxxxx
+;;!       ---------
+;;!! {value: key for (key, value) in d1.items()}
+;;!          ^^^
+;;!        xxxxx
+;;!   ----------
+;;!! def func(value: str = ""):
+;;!                        ^^
+;;!                     xxxxx
+;;!           ---------------
+(
+  (_
+    (_) @_.leading.start.endOf
+    .
+    value: (_) @value @_.leading.end.startOf
+  ) @_.domain
+  (#not-type? @_.domain subscript)
+)
+
+;;!! return 1
+;;!         ^
+;;!        xx
+;;!  --------
+;;
+;; NOTE: in tree-sitter, both "return" and the "1" are children of `return_statement`
+;; but "return" is anonymous whereas "1" is named node, so no need to exclude explicitly
```

Comment:
This comment might be overkill. We leverage this kind of assumption all over the place. For example, I'm leveraging this kind of assumption in my leading matchers above, where I rely on the fact that `=` is anonymous whereas the name / type before it is not